src/puppetbridge.ts

```diff
@@ -211,6 +229,37 @@ export class PuppetBridge extends EventEmitter {
 		this.botProvisioner = new BotProvisioner(this);
 		this.provisioningAPI = new ProvisioningAPI(this);
 
+		if (this.config.metrics.enabled) {
+			prometheus.collectDefaultMetrics();
+			const metricsServer = express();
```

Comment:
why don't we re-use our existing express instance, so that we only have to have one single http server running?

Reply:
As far as i could see, there is no simple way to listen on different ports with the same express instance - it seems to be relatively common to have metrics on a seperate port from the "main service", but i haven't looked into the common way of archieving that.

Reply:
BUMP